__tests__/endpoints.test.js

```diff
@@ -25,6 +26,57 @@ describe('GET /api', () => {
     })
 })
 
+describe('GET /api/articles/:article_id', () => {
+    describe('200', () => {
+        test('200: responds with an article with the requested article_id', () => {
+            return request(app)
+            .get('/api/articles/1')
+            .expect(200)
+            .then(({body}) => {
+                expect(Object.keys(body)).toEqual([
+                    'article_id',
+                    'title',
+                    'topic',
+                    'author',
+                    'body',
+                    'created_at',
+                    'votes',
+                    'article_img_url'
```

Comment:
Just a note- later on this endpoint will be adding a further key. I'm pretty sure that this test will still pass the way you've done it, but if not, you may have to come back and change this to a more flexible matcher later 😄

Reply:
Thanks for the tip, I'll keep that in mind ;)